code/settings.gradle

```diff
@@ -28,5 +28,9 @@ dependencyResolutionManagement {
   maven { url = uri("https://jitpack.io") }
  }
 }
- include ':campaignclassic', ':testapp'
-rootProject.name = "aepsdk-campaignclassic-android"
\ No newline at end of file
+include ':campaignclassic', ':testapp'
+rootProject.name = "aepsdk-campaignclassic-android"
+// todo remove when snapshot is available
+include ':notificationbuilder'
+project(':notificationbuilder').projectDir =
```

Comment:
this import will only work if the notification builder code is present on the local machine, right? should we make the notification builder code a submodule? this doesn't matter though if we switch to jitpack / snapshot soon.

Reply:
Yes I will change this once we have a final version of dev build available